static/js/publisher/release/components/devmodeRevision.js

```diff
@@ -3,14 +3,24 @@ import PropTypes from "prop-types";
 
 import { isInDevmode } from "../helpers";
 
-export default function DevmodeRevision({ revision, showTooltip }) {
+export default function DevmodeRevision({
```

Comment:
Now I realised that the name of this component is confusing.

It seems to be more like "RevisionLabel" or something... Isn't specific to devmode revisions at all...